src/components/orders/OrdersUserDetailsTable/index.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Story, Meta } from '@storybook/react/types-6-0'
+import OrderUserDetailsTable, { Props as OrdersUserTableProps } from '.'
+import { add, sub } from 'date-fns'
+import BigNumber from 'bignumber.js'
+import { GlobalStyles, ThemeToggler, Router, NetworkDecorator } from 'storybook/decorators'
+
+import { Order } from 'api/operator'
+import { RICH_ORDER, TUSD, WETH } from '../../../../test/data'
+
+export default {
+  title: 'orders/OrdersUserDetailsTable',
+  decorators: [Router, GlobalStyles, NetworkDecorator, ThemeToggler],
+  component: OrderUserDetailsTable,
+} as Meta
+
+const orderBuy: Order = {
+  ...RICH_ORDER,
```

Comment:
It's possible the inverted price issue I mentioned in the other comment to be related to this bug https://github.com/gnosis/gp-ui/blob/c89c25dc5d7fa612c8cd01c3d5046860a355f34f/test/data/operator.ts#L21-L52
Ivica fixed it on his PR. The sell/buy tokens are inverted in the `RICH_ORDER`

Reply:
It was a really a trouble maker :smile: